index.js

```diff
@@ -285,7 +292,7 @@ ExtractTextPlugin.prototype.apply = function(compiler) {
 				callback();
 			}.bind(this));
 		}.bind(this));
-		compilation.plugin("additional-assets", function(callback) {
+		compilation.plugin("before-chunk-assets", function() {
```

Comment:
@cletusw also, because of changing `additional-assets` to `before-chunk-assests` webpack does not produce source maps anymore :(
I am not sure how to fix this..